dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -48,24 +49,22 @@
 import static io.swagger.api.impl.ToolsImplCommon.WORKFLOW_PREFIX;
 
 public final class ZenodoHelper {
-    private static final Logger LOG = LoggerFactory.getLogger(WorkflowResource.class);
+    private static final Logger LOG = LoggerFactory.getLogger(ZenodoHelper.class);
 
     private ZenodoHelper() {
     }
 
     /**
      * Register a Zenodo DOI for the workflow version
-     * @param zenodoClient
-     * @param dockstoreGA4GHBaseUrl The baseURL for GA4GH tools endpoint (e.g. "http://localhost:8080/api/api/ga4gh/v2/tools/")
-     * @param dockstoreUrl URL for Dockstore (e.g. https://dockstore.org)
+     * @param zenodoClient Client for interacting with Zenodo server
      * @param workflowUrl Dockstore workflow URL (e.g. https://dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl)
      * @param workflow    workflow for which DOI is registered
      * @param workflowVersion workflow version for which DOI is registered
-     * @param entryVersionHelper code for interacting with the files of versions, we use zip file creation methods
+     * @param workflowResource code for interacting with the files of versions(we use zip file creation methods) and aliases
+     * @param user user authenticated to issue a DOI for the workflow
      */
-    public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String dockstoreGA4GHBaseUrl,
-            String dockstoreUrl, String workflowUrl, Workflow workflow,
-            WorkflowVersion workflowVersion, EntryVersionHelper entryVersionHelper) {
+    public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String workflowUrl, Workflow workflow,
```

Comment:
Passing the workflowResource makes this really hard to use by any other class except for workflowResource, which brings up the question as to why this method isn't simply in WorkflowResource.

Is it possible to re-use this method for Tools later?

Reply:
We can modify ZenodoHelper to use Entry and Version instead of Workflow and WorkflowVersion in another PR to support generation of DOIs for Tools